docs/common/object.ldoc

```diff
@@ -1,28 +1,30 @@
 
---- Disconnect to a signal.
--- @tparam string name The name of the signal
--- @tparam function func The callback that should be disconnected
+--- Disconnect a signal.
```

Comment:
Perhaps "disconnect from a signal"? Is `property::foo` the signal, or is the connection between `property::foo` and the provided callback function the signal? I'd say the former, so `from a signal` would be correct.